atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/atomic/ConsensusForgettingStore.java

```diff
@@ -32,7 +31,23 @@
  *   3. Once (1) or (2) is successful, get will never return an Optional.empty() and will always return a consistent
  *   value until a put occurs.
  */
-public interface ConsensusForgettingStore {
+public interface ConsensusForgettingStore extends ConsensusForgettingStoreReader {
```

Comment:
nit: a "store reader" seems to be a separate concept from a store (and inheritance normally expresses an is-a relationship). Maybe the other should be "ReadableConsensusForgettingStore`?

Reply:
great catch! i started off with a different design and forgot to change the name.